colorizer_data/bin/example_scripts/convert_emt_migration_data.py

```diff
@@ -109,42 +149,22 @@ def make_frames(
     Generate the images and bounding boxes for each time step in the dataset.
     """
     nframes = len(grouped_frames)
+    total_objects = get_total_objects(grouped_frames)
     logging.info("Making {} frames...".format(nframes))
 
     bounds_arr = make_bounding_box_array(grouped_frames)
 
-    for group_name, frame in grouped_frames:
-        start_time = time.time()
-
-        # Get the path to the segmented zstack image frame from the first row (should be the same for
-        # all rows in this group, since they are all on the same frame).
-        row = frame.iloc[0]
-        frame_number = row[TIMES_COLUMN]
-        # Flatten the z-stack to a 2D image.
-        aics_image = get_image_from_row(row)
-        seg2d = aics_image.get_image_data("YX", S=0, T=0, C=0)
-
-        # Scale the image and format as integers.
-        seg2d = scale_image(seg2d, scale)
-        seg2d = seg2d.astype(np.uint32)
-
-        # Remap the frame image so the IDs are unique across the whole dataset.
-        seg_remapped, lut = remap_segmented_image(
-            seg2d,
-            frame,
-            OBJECT_ID_COLUMN,
-        )
-
-        writer.write_image(seg_remapped, frame_number)
-        update_bounding_box_data(bounds_arr, seg_remapped)
-
-        time_elapsed = time.time() - start_time
-        logging.info(
-            "Frame {} finished in {:5.2f} seconds.".format(
-                int(frame_number), time_elapsed
+    with multiprocessing.Manager() as manager:
+        bounds_arr = manager.Array("i", [0] * int(total_objects * 4))
```

Comment:
I'm a little confused about `bounds_arr`. Is line 155 still necessary if it's set again here, in the context of the manager?

Reply:
Oh, you're right, I missed removing that. Thank you for catching it!